app/assertions/footer_assert.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class FooterAssertion
+    include Capybara::DSL
+    include RSpec::Matchers
+  
+    def initialize
+      @footer = FooterMethods.new
+      @login = LoginMethods.new
+    end
+  end
+
+
+  def assert_shipping_link_works
+    @footer.click_shipping
+    expect(page).to have_current_path('/pages/shipping-payments')
+  end
+
+  def assert_returns_link_works
+    @footer.click_returns
```

Comment:
Remove the click method from here. The click method should be called on the _Footer_spec.rb_ file, in order the assertion can be re-used in other possible verification.